src/Core/Configurations/RuntimeConfigValidator.cs

```diff
@@ -345,7 +346,7 @@ public void ValidateEntitiesDoNotGenerateDuplicateQueriesOrMutation(RuntimeEntit
             }
             else
             {
-                // For entities (table/view) that have graphQL exposed, two queries and three mutations would be generated.
+                // For entities (table/view) that have graphQL exposed, two queries and four mutations would be generated.
```

Comment:
only four mutations for CosmosDB though. So you need to make that distinction in this comment.